packages/reporters/cli/src/index.js

```diff
@@ -1,8 +1,10 @@
 // @flow strict-local
 
 // $FlowFixMe This is fine.
-const isTTY = process.stdout.isTTY;
+// const isTTY = process.stdout.isTTY;
```

Comment:
Can this be reverted back?

Reply:
Kind of not really. It breaks with the servers for some reason. So that would need to get fixed, but probably not in this PR